WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -139,14 +143,30 @@ + (void)setShouldWaitForQuiescence:(BOOL)value
 
 #pragma mark Private
 
-+ (NSRange)bindingPortRangeFromArguments
++ (NSString*)valueFromArguments: (NSArray<NSString *> *)arguments forKey: (NSString*)key
 {
-  NSArray *arguments = NSProcessInfo.processInfo.arguments;
-  NSUInteger index = [arguments indexOfObject:@"--port"];
+  NSUInteger index = [arguments indexOfObject:key];
   if (index == NSNotFound || index == arguments.count - 1) {
-    return NSMakeRange(NSNotFound, 0);
+    return @"";
```

Comment:
I'd rather return nil if nothing is found

Reply:
There is `NS_ASSUME_NONNULL_BEGIN` macro in the header file. Returning `nil` here will cause a warning and will create inconsistency, potentially leading to a bug in the future.

Reply:
feel free to mark the result as nullable